publicdb/status_display/status.py

```diff
@@ -0,0 +1,68 @@
+import datetime
+
+from ..inforecords.models import Station, Pc
+from ..histograms.models import Summary
+
```

Comment:
Remove the blank line (only one blank line between imports and variables/constants).
Reorder the imports alphabetically (i.e. `h` before `i`), you can also run `isort` locally and it should propose that fix.

Reply:
Ok, done in bc9e8c7, thanks! That's pretty strict, ;-).